pkg/resources/dynamic_table.go

```diff
@@ -61,7 +61,6 @@ var dynamicTableShema = map[string]*schema.Schema{
 		Type:        schema.TypeString,
 		Required:    true,
 		Description: "The warehouse in which to create the dynamic table.",
-		ForceNew:    true,
```

Comment:
Please add the `warehouse` param change in the `TestAcc_DynamicTable_basic` test.